varats-core/varats/experiment/workload_util.py

```diff
@@ -92,8 +93,21 @@ def workload_commands(
         )
     ]
 
+    # Filter commands that have required args set.
+    extra_options = set(get_extra_config_options(project))
+
+    def requires_filter(prj_cmd: ProjectCommand) -> bool:
+        if hasattr(prj_cmd.command, "requires") and prj_cmd.command.requires:
+            args = set(prj_cmd.command._args).union(extra_options)
+            return bool(args.intersection(prj_cmd.command.requires))
```

Comment:
Is this correct? As far as I can see, this only checks if there is an intersection, but not if all required commands are contained. The following should do the trick:
```suggestion
            return prj.command.requires.issubset(args)
```

Reply:
That is intended behavior. It must be in the requires set. Otherwise I would have to duplicate commands for my use case: https://github.com/se-sic/VaRA-Tool-Suite/blob/37793ec0aff6def2c867bc69dde1c71be3d31c33/varats/varats/projects/c_projects/xz.py#L144
If someone needs the all required functionality, we can still add a `requires_all` field.

Reply:
I see, that is something we should definitely consider here.
It would probably be a good idea to have both, a `requires_all` and a `requires_any`.
The check then should only succeed if both are fulfilled.